Assets/Plugins/LeapMotion/Core/Scripts/LeapServiceProvider.cs

```diff
@@ -461,7 +478,19 @@ public enum PhysicsExtrapolationMode {
         return;
       }
 
-      _leapController.ClearPolicy(Controller.PolicyFlag.POLICY_DEFAULT);
+      if (_trackingOptimization == TrackingOptimizationMode.Desktop) {
+        _leapController.ClearPolicy(Controller.PolicyFlag.POLICY_DEFAULT);
+        _leapController.ClearPolicy(Controller.PolicyFlag.POLICY_OPTIMIZE_SCREENTOP);
+        _leapController.ClearPolicy(Controller.PolicyFlag.POLICY_OPTIMIZE_HMD);
+      } else if (_trackingOptimization == TrackingOptimizationMode.ScreenTop) {
+        _leapController.ClearPolicy(Controller.PolicyFlag.POLICY_DEFAULT);
+        _leapController.ClearPolicy(Controller.PolicyFlag.POLICY_OPTIMIZE_HMD);
+        _leapController.SetPolicy  (Controller.PolicyFlag.POLICY_OPTIMIZE_SCREENTOP);
+      } else if (_trackingOptimization == TrackingOptimizationMode.HMD) {
+        _leapController.ClearPolicy(Controller.PolicyFlag.POLICY_DEFAULT);
+        _leapController.ClearPolicy(Controller.PolicyFlag.POLICY_OPTIMIZE_SCREENTOP);
+        _leapController.SetPolicy  (Controller.PolicyFlag.POLICY_OPTIMIZE_HMD);
```

Comment:
Controller.PolicyFlag doesn't contain a definition for Controller.PolicyFlag.POLICY_OPTIMIZE_SCREENTOP. I assume it just needs to be added as a new flag at 1 << 8?